docs/_user_guide/best_practices_optimization.md

```diff
@@ -1,306 +1,354 @@
+Python 3.10.12 (main, Sep 11 2024, 15:47:36) [GCC 11.4.0] on linux
 Type "help", "copyright", "credits" or "license" for more information.
->>> from gurobipy import *
+>>>
+```
+
+Ensure that you can import `gurobipy`by running the following command:
+
+```title="Terminal Command"
+from gurobipy import *
 ```
 
-Exit the REPL.
- 
-If you are done working interactively and want to deactivate the virtual environment and remove all modules, run:
-```bash
-$ deactivate
-$ module purge
+To exit the REPL, type:
+
+```title="Terminal Command"
+exit()
 ```
 
+When you are finished working interactively and want to deactivate the virtual environment and unload all modules, run the following commands:
 
-The environment is now set up to run your Python scripts that import `gurobipy` on interactive Yen nodes.
-Remember that the `module load` command and virtual environment activation is only active in the current shell.
+```title="Terminal Command"
+deactivate
+module purge
+```
 
-**Important:** You need to load the `gurobipy3` module and activate your `venv` environment every time you login to the Yens
-before running the interactive Python scripts that use `gurobipy` package. 
+The environment is now set up to run your Python scripts that import `gurobipy` on the interactive Yen nodes. Keep in mind that the `module load` command and virtual environment activation are only active in the current shell.
 
+!!! Important
+    You need to load the `gurobipy3` module and activate your `venv` environment every time you log in to the Yens before running the interactive Python scripts that use the `gurobipy` package.
 
-### Running Gurobi in R
-Similar to running Gurobi in Python, Gurobi R package is also installed and available system-wide to use on the Yens. 
-You do not need to install anything into your user R library. 
+#### Running Gurobi in R
 
-**Environment Setup**
+Similar to running Gurobi in Python, Gurobi R package is also installed and available system-wide to use on the Yens. You do not need to install anything into your user R library.
 
-To use Gurobi software with R, you simply need to load both modules:
+##### Environment Setup
 
-```bash
-$ ml gurobi R
+To use Gurobi software with R, simply load both modules:
+
+```title="Terminal Command"
+ml gurobi R/4.2
 ```
 
-List loaded modules:
-```bash
-$ ml 
+To list the currently loaded modules, use:
+
+```title="Terminal Command"
+ ml
+```
+```{ .yaml .no-copy title="Terminal Output" }
 Currently Loaded Modules:
-  1) gurobi/10.0.0   2) rstudio/2022.07.2+576   3) R/4.2.1
+  1) gurobi/10.0.0   2) R/4.2.0
 ```
 
 Launch interactive R:
-```bash
-$ R
+
+```title="Terminal Command"
+ R
 ```
 
 Then you can load the `gurobi` R package:
 
-```R
-R
-
-R version 4.2.1 (2022-06-23) -- "Funny-Looking Kid"
-Copyright (C) 2022 The R Foundation for Statistical Computing
+```{ .yaml .no-copy title="Terminal Output" }
+R version 4.3.0 (2023-04-21) -- "Already Tomorrow"
```

Comment:
I think you have to load R/4.2. But here is the output for R 4.3. It might be a bit confusing